### PR TITLE
robot_systemd: 0.1.2-1 in 'melodic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by LucidOne**
Increasing version of package(s) in repository `robot_systemd` to `0.1.2-1`:

- upstream repository: https://github.com/LucidOne/robot_systemd.git
- release repository: https://github.com/LucidOne-Release/robot_systemd.git
- distro file: `melodic/distribution.yaml`
- bloom version: `0.9.0`
- previous version for package: `null`

## robot_systemd

```
* Removed pre-emptive conflict
* Added TasksMax=infinity
* Added KillSignal=SIGINT
* Added documentation
```

### CODE REVIEW

File: melodic/distribution.yaml
Comment (clalancette) [inline melodic/distribution.yaml]: Looking at the upstream sources, there are a couple of license things that I think would be best cleared up before we merge this.

1.  The package.xml lists both MIT and Apache.  Is it really under both of them?  Also, "Apache" as a license probably isn't specific enough; see the list https://opensource.org/licenses/alphabetical .
1.  It would be better if there was a top-level LICENSE file in the repository, which would clarify some of these questions.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: melodic/distribution.yaml
Comment (LucidOne) [reply melodic/distribution.yaml]: Thanks for the  feedback, I'll get an update out in a bit.

This covers  dual licensing
https://news.ycombinator.com/item?id=15956857

Comment (clalancette) [reply melodic/distribution.yaml]: > This covers dual licensing
> https://news.ycombinator.com/item?id=15956857

Dual-licensed packages are allowed, but it is unusual to see it with a package that is so young and has commits from only one committer.  If you want to make it maximally applicable, you could just leave it as MIT or BSD, which can be combined with most other things.

But the dual-license thing isn't my major concern; as I said, we'll take it that way.  My concerns are that "Apache" isn't specific enough, and there are no top-level LICENSE files.

**Comment by clalancette** [comment]
No response in over a month.  I'm going to close this out, but feel free to fix the issues pointed out during review and re-open.  Thanks for the contribution.

**Pull request closed without merging**
